javascript/node/selenium-webdriver/lib/webdriver.js

```diff
@@ -1171,6 +1171,12 @@ class Options {
 
   /**
    * Schedules a command to delete all cookies visible to the current page.
+   * 
+   * __Sample Usage:__
+   * 
+   * //delete all the cookies
```

Comment:
These lines need to be indented 4 spaces past the * to be formatted as a code block in the generated docs. To check your formatting, run `node javascript/node/gendocs.js`

Output will be generated in `build/javascript/node/selenium-webdriver-docs`